CMakeLists.txt

```
@@ -1033,6 +1030,9 @@ endif ()
 # --with-openssl (which may be common).
 include_directories(BEFORE ${OPENSSL_INCLUDE_DIR})
 
+# Propagate non-standard OpenSSL includes to dynamic plugins.
+target_include_directories(zeek_dynamic_plugin_base INTERFACE "${OPENSSL_INCLUDE_DIR}")
```

Could use this instead to suppress warnings from OpenSSL headers and make sure they are searched before other include paths:

```suggestion
target_include_directories(zeek_dynamic_plugin_base SYSTEM BEFORE INTERFACE "${OPENSSL_INCLUDE_DIR}")
```

Doesn't really matter that much, though. I can't remember ever seeing warnings from OpenSSL headers. So approving anyways.